src/components/Reports/LocationSection.tsx

```diff
@@ -133,6 +185,43 @@ export default function LocationSection(props: LocationSectionProps): JSX.Elemen
     }
   };
 
+  const getLivePlants = () => {
+    return latestObservation?.species.reduce((acc, sp) => (acc = acc + sp.permanentLive), 0);
+  };
+
+  const getDeadPlants = () => {
+    return latestObservation?.species.reduce((acc, sp) => (acc = acc + sp.cumulativeDead), 0);
+  };
+
+  const getNumberOfPlots = () => {
+    return latestObservation?.plantingZones.flatMap((pz) =>
+      pz.plantingSubzones.flatMap((subzone) => subzone.monitoringPlots)
+    ).length;
+  };
+
+  const getMarkedAsComplete = () => {
+    if (plantingSite) {
+      const totalArea = plantingSite.areaHa ?? 0;
+      const totalPlantedArea =
+        plantingSite?.plantingZones
+          ?.flatMap((zone) => zone.plantingSubzones)
+          ?.reduce((prev, curr) => (curr.plantingCompleted ? +curr.areaHa + prev : prev), 0) ?? 0;
+      const percentagePlanted = totalArea > 0 ? Math.round((totalPlantedArea / totalArea) * 100) : 0;
+      return `${percentagePlanted}%`;
+    }
+    return '0%';
+  };
+
+  const getPlantingDensityForZones = () => {
+    if (plantingSite && plantingDensity) {
+      const zoneNameWithDensities = plantingSite.plantingZones?.map(
+        (zone) => `${zone.name}: ${plantingDensity[zone.name]}`
+      );
+      return zoneNameWithDensities?.join('\r\n') || '';
+    }
+    return '';
+  };
+
```

Comment:
if it makes sense, use `useMemo` for each of these? Will make the page less busy, it is a busy page.